app/topics/show-topic/show-topic.component.html

```diff
@@ -44,7 +44,8 @@
       <button md-raised-button color="primary" [routerLink]="[ '/topics/edit', topic.id ]">
         {{ "edit details" | translate }}
       </button>
-      <button md-raised-button color="primary">{{ "edit content" | translate }}</button>
+      <button md-raised-button color="primary">{{ 'edit content' | translate }}</button>
+      <button md-raised-button color="warn" [routerLink]="[ '/topics/delete', topic.id ]">{{ 'delete topic' | translate }}</button>
```

Comment:
It would be nice, if the button is only visible when I am allowed to delete the topic.
So the button is visible if it is a topic created by me, or I am an admin.